python/ctsm/site_and_regional/run_neon.py

```diff
@@ -175,7 +175,7 @@ def main(description):
     cesmroot = path_to_ctsm_root()
     # Get the list of supported neon sites from usermods
     valid_neon_sites = glob.glob(
-        os.path.join(cesmroot, "cime_config", "usermods_dirs", "NEON", "[!d]*")
+        os.path.join(cesmroot, "cime_config", "usermods_dirs", "NEON", "[!Fd]*")
```

Comment:
What does the F do?

Reply:
This says don't get cases that start with either F or d. It was confusing in FATES cases.

This does mean if we add cases that do start with it, it will fail in possibly a weird way.

So add a comment about that...